# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# Funda Wang <fundawang@linux.net.cn>, 2005.
#
# YunQiang Su <wzssyqa@gmail.com>, 2012.
#
#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
msgstr ""
"Project-Id-Version: update-manager HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2021-07-19 13:55-0700\n"
"PO-Revision-Date: 2019-10-26 18:45+0000\n"
"Last-Translator: Xu Zhen <Unknown>\n"
"Language-Team: Chinese (simplified) <i18n-zh@googlegroups.com>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:48+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "%s 的服务器"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "主服务器"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "自定义服务器"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "卸载状态异常的软件包"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"软件包“%s”处于不一致的状态，需要重新安装，但是没有找到对应的存档。您希望现在"
"删除这个软件包以进行下一步吗？"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "服务器可能已过载"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "破损的软件包"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"您的系统包含有本软件不能修复的破损软件包，在您继续前请先用新立得或者 apt-get "
"修复它们。"

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "破损的软件包"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "很可能发生了一个传输问题，请稍后重试。"

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "无法计算升级"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "一些软件包认证出错"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"一些软件包无法通过签名验证。这可能是暂时的网络问题，您可以在稍后再试。以下是"
"未认证软件包的列表。"

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "软件包“%s”标记为可移除，但它已在移除黑名单中。"

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "必要的软件包“%s”被标记为移除。"

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "尝试安装黑名单版本“%s”"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "无法安装 '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"无法安装所需的软件包。请在终端使用“ubuntu-bug ubuntu-release-upgrader-core”将"
"其报为 bug。"

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "无法猜出元软件包"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"您的系统没有包含 %s 或 %s 包，不可能检测到您正在运行的 Ubuntu 版本。\n"
"请先使用 synaptic 安装上述软件包之一或 apt-get 。"

#: ../DistUpgrade/DistUpgradeController.py:112
msgid "Reading cache"
msgstr "正在读取缓存"

#: ../DistUpgrade/DistUpgradeController.py:219
msgid "Unable to get exclusive lock"
msgstr "无法获得排它锁"

#: ../DistUpgrade/DistUpgradeController.py:220
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"这通常意味着另一个软件包管理程序(如 apt-get 或 aptitude)正在运行。请先关闭那"
"个程序。"

#: ../DistUpgrade/DistUpgradeController.py:278
msgid "Upgrading over remote connection not supported"
msgstr "不支持通过远程连接升级"

#: ../DistUpgrade/DistUpgradeController.py:279
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"您正在通过远程 SSH 升级，而前端程序不支持这种方式。请尝试在文本模式下通过 "
"'do-release-upgrade' 命令进行升级。\n"
"\n"
"现在将退出升级。请试试不使用 ssh 的方式。"

#: ../DistUpgrade/DistUpgradeController.py:293
msgid "Continue running under SSH?"
msgstr "继续在 SSH 下执行？"

#: ../DistUpgrade/DistUpgradeController.py:294
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"此会话似乎是在 SSH 下运行。目前不推荐通过 SSH 执行升级，因为升级失败时较难恢"
"复。\n"
"\n"
"如果您选择继续，将在 '%s' 端口上建立额外的 SSH 守护进程。\n"
"您想要继续吗？"

#: ../DistUpgrade/DistUpgradeController.py:308
msgid "Starting additional sshd"
msgstr "正在启用额外的 ssh 守护进程"

#: ../DistUpgrade/DistUpgradeController.py:309
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"为了在失败时更容易恢复，将在端口“%s”开启一个额外的 ssh 守护进程。如果当前运行"
"的 ssh 发生错误，您仍能够通过该额外的 ssh 进行连接。\n"

#: ../DistUpgrade/DistUpgradeController.py:317
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"如果您运行了一个防火墙，可能需要临时打开这个端口。这可能有些危险，因此没有自"
"动进行这个操作。您可以通过类似这样的命令打开端口：\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:395
#: ../DistUpgrade/DistUpgradeController.py:419
msgid "Can not upgrade"
msgstr "无法升级"

#: ../DistUpgrade/DistUpgradeController.py:396
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "此工具不支持从 '%s' 到 ‘%s' 的升级。"

#: ../DistUpgrade/DistUpgradeController.py:420
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:449
#, python-format
msgid "Can not write to '%s'"
msgstr "无法写入 '%s'"

#: ../DistUpgrade/DistUpgradeController.py:450
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"无法写入您的系统目录 %s ，升级无法继续。\n"
"请确保系统目录可写。"

#: ../DistUpgrade/DistUpgradeController.py:617
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr "%s 版本尚在开发阶段，不适合普通用户使用"

#: ../DistUpgrade/DistUpgradeController.py:717
#, python-format
msgid "disabled on upgrade to %s"
msgstr "已禁止升级到 %s"

#: ../DistUpgrade/DistUpgradeController.py:776
msgid "No valid sources.list entry found"
msgstr "找不到有效的 sources.list 项"

#: ../DistUpgrade/DistUpgradeController.py:777
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""
"扫描您的仓库信息时找不到有关 %s 的入口。\n"
"\n"
"升级可能无法成功进行。\n"
"\n"
"仍想继续吗？"

#: ../DistUpgrade/DistUpgradeController.py:787
msgid "No valid mirror found"
msgstr "未找到可用的镜像"

#: ../DistUpgrade/DistUpgradeController.py:788
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"在扫描您的仓库信息时，未发现升级可用的镜像条目。可能是由于您运行一个内部镜像"
"或者镜像信息过时。\n"
"\n"
"您要重写您的 ‘sources.list’ 文件？选择“是”，将更新所有从'%s'到'%s'的条目。\n"
"选择“否”取消升级。"

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:808
msgid "Generate default sources?"
msgstr "生成默认的源？"

#: ../DistUpgrade/DistUpgradeController.py:809
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"扫描“sources.list”后未发现用于“%s”的可用项。\n"
"\n"
"是否为“%s”添加默认项？如果选择“否”，将会取消升级。"

#: ../DistUpgrade/DistUpgradeController.py:844
#: ../DistUpgrade/DistUpgradeController.py:850
msgid "Repository information invalid"
msgstr "仓库信息无效"

#: ../DistUpgrade/DistUpgradeController.py:845
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr "更新软件源时返回了无效文件，已启动错误报告进程。"

#: ../DistUpgrade/DistUpgradeController.py:851
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"升级软件仓库信息时得到了无效的文件。如果您想报告这个错误，请安装 apport 软"
"件，然后执行 apport-bug ubuntu-release-upgrader 命令。"

#: ../DistUpgrade/DistUpgradeController.py:860
msgid "Third party sources disabled"
msgstr "第三方源被禁用"

#: ../DistUpgrade/DistUpgradeController.py:861
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"您的 sources.list 中的一些第三方源被禁用。您可以在升级后用\"软件源\"工具或包"
"管理器来重新启用它们。"

#: ../DistUpgrade/DistUpgradeController.py:903
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "软件包存在冲突"

#: ../DistUpgrade/DistUpgradeController.py:906
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"软件包“%s”处于冲突状态，需要重新安装， 但是没能找到它的存档。请重新手动安装这"
"个软件包或者将其从系统中删除。"

#: ../DistUpgrade/DistUpgradeController.py:969
msgid "Error during update"
msgstr "升级时出错"

#: ../DistUpgrade/DistUpgradeController.py:970
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr "升级过程中出错。这通常是一些网络问题，请检查您的网络连接后再试"

#: ../DistUpgrade/DistUpgradeController.py:1013
#: ../DistUpgrade/DistUpgradeController.py:1018
msgid "EFI System Partition (ESP) not usable"
msgstr "EFI 系统分区 (ESP) 无法使用"

#: ../DistUpgrade/DistUpgradeController.py:1014
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""
"您的 EFI 系统分区 (ESP) 没有挂载在 /boot/efi，请确保其正确配置后再试一次。"

#: ../DistUpgrade/DistUpgradeController.py:1019
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""
"挂载到 /boot/efi 的 EFI 系统分区 (ESP) 无法写入，请将分区挂载为可读后重试。"

#: ../DistUpgrade/DistUpgradeController.py:1026
msgid "Not enough free disk space"
msgstr "磁盘空间不足"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1029
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""
"此次升级被迫中止。磁盘  '{str_dir}' 总共需要有 {str_total} 可用空间才能进行更"
"新。请再腾出 '{str_dir}' 上至少 {str_needed} 磁盘空间。{str_remedy}"

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1034
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr "使用“sudo apt clean”删除前一次安装的临时软件包。"

#: ../DistUpgrade/DistUpgradeController.py:1036
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""
"您可以使用“sudo apt autoremove”删除旧内核，并您也可以在 /etc/initramfs-tools/"
"initramfs.conf 中设置 COMPRESS=xz 以减少您的 initramfs 的大小。"

#: ../DistUpgrade/DistUpgradeController.py:1041
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr "使用“sudo apt-get clean”清空您的回收站并删除前一次安装的临时软件包。"

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid "Reboot to clean up files in /tmp."
msgstr "重启以清空 /tmp 中的文件。"

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1083
#: ../DistUpgrade/DistUpgradeController.py:1925
#: ../DistUpgrade/DistUpgradeController.py:1977
msgid "Calculating the changes"
msgstr "正在计算变更"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1127
msgid "Do you want to start the upgrade?"
msgstr "您要开始升级么？"

#: ../DistUpgrade/DistUpgradeController.py:1146
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1218
msgid "Upgrade canceled"
msgstr "升级已取消"

#: ../DistUpgrade/DistUpgradeController.py:1219
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr "升级将会取消，系统会恢复到原始状态。您可以在之后继续升级。"

#: ../DistUpgrade/DistUpgradeController.py:1225
#: ../DistUpgrade/DistUpgradeController.py:1370
msgid "Could not download the upgrades"
msgstr "无法下载升级包"

#: ../DistUpgrade/DistUpgradeController.py:1226
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"已中止升级。请检查您的互联网连接或安装媒体并重试。所有已下载的文件都已保存。"

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1319
#: ../DistUpgrade/DistUpgradeController.py:1358
#: ../DistUpgrade/DistUpgradeController.py:1474
msgid "Error during commit"
msgstr "确认时出错"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1321
#: ../DistUpgrade/DistUpgradeController.py:1360
#: ../DistUpgrade/DistUpgradeController.py:1519
msgid "Restoring original system state"
msgstr "正在恢复原始系统状态"

#: ../DistUpgrade/DistUpgradeController.py:1322
#: ../DistUpgrade/DistUpgradeController.py:1337
#: ../DistUpgrade/DistUpgradeController.py:1361
msgid "Could not install the upgrades"
msgstr "无法安装升级"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1327
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"更新已取消。您的系统可能处在不稳定状态。正在恢复 (dpkg --configure -a)。"

#: ../DistUpgrade/DistUpgradeController.py:1332
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"请将问题报告至 http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-"
"upgrader/+filebug 并将 /var/log/dist-upgrade/ 文件作为附件上传到问题报告。\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1371
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr "更新已取消。请检查您的因特网连接或安装媒体，然后再试一遍。 "

#: ../DistUpgrade/DistUpgradeController.py:1399
msgid "Searching for obsolete software"
msgstr "正在搜索废弃的软件"

#: ../DistUpgrade/DistUpgradeController.py:1462
msgid "Remove obsolete packages?"
msgstr "删除陈旧的软件包？"

#: ../DistUpgrade/DistUpgradeController.py:1463
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "保持(_K)"

#: ../DistUpgrade/DistUpgradeController.py:1463
msgid "_Remove"
msgstr "删除(_R)"

#: ../DistUpgrade/DistUpgradeController.py:1475
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "清理时出现问题。更多信息请查看以下消息。 "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1557
msgid "Required depends is not installed"
msgstr "需要的依赖关系未安装"

#: ../DistUpgrade/DistUpgradeController.py:1558
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "需要的依赖关系“%s”未安装 。 "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1791
#: ../DistUpgrade/DistUpgradeController.py:1875
msgid "Checking package manager"
msgstr "正在检查软件包管理器"

#: ../DistUpgrade/DistUpgradeController.py:1797
#: ../DistUpgrade/DistUpgradeController.py:1803
msgid "Preparing the upgrade failed"
msgstr "升级准备失败"

#: ../DistUpgrade/DistUpgradeController.py:1798
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr "更新时的系统准备失败，已启动错误报告进程。"

#: ../DistUpgrade/DistUpgradeController.py:1804
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"更新时的系统准备失败。如果您想报告这个错误，请安装 apport 软件，然后执行 "
"apport-bug ubuntu-release-upgrader 命令。"

#: ../DistUpgrade/DistUpgradeController.py:1821
#: ../DistUpgrade/DistUpgradeController.py:1831
msgid "Getting upgrade prerequisites failed"
msgstr "准备升级失败"

#: ../DistUpgrade/DistUpgradeController.py:1822
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"系统不能满足升级先决条件。现在会中止升级并恢复原来的系统状态。\n"
"\n"
"此外，错误报告进程也将启动。"

#: ../DistUpgrade/DistUpgradeController.py:1832
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"系统不能满足升级先决条件。现在会中止升级并恢复原来的系统状态。\n"
"\n"
"如果您想报告这个错误，请安装 apport 软件，然后执行 apport-bug ubuntu-release-"
"upgrader 命令。"

#: ../DistUpgrade/DistUpgradeController.py:1866
msgid "Updating repository information"
msgstr "正在更新软件仓库信息"

#: ../DistUpgrade/DistUpgradeController.py:1902
msgid "Invalid package information"
msgstr "无效的软件包信息"

#: ../DistUpgrade/DistUpgradeController.py:1903
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1932
#: ../DistUpgrade/DistUpgradeController.py:2038
msgid "Fetching"
msgstr "正在获取"

#: ../DistUpgrade/DistUpgradeController.py:1939
#: ../DistUpgrade/DistUpgradeController.py:1962
#: ../DistUpgrade/DistUpgradeController.py:1992
#: ../DistUpgrade/DistUpgradeController.py:2042
msgid "Upgrading"
msgstr "正在升级"

#: ../DistUpgrade/DistUpgradeController.py:1941
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1942
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr "无法完成升级，因为升级过程中出现错误。"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1965
#: ../DistUpgrade/DistUpgradeController.py:1981
msgid "Upgrade incomplete"
msgstr "未完成升级"

#: ../DistUpgrade/DistUpgradeController.py:1966
#: ../DistUpgrade/DistUpgradeController.py:1982
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2044
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2061
msgid "Upgrade complete"
msgstr "升级完成"

#: ../DistUpgrade/DistUpgradeController.py:1998
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2051
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "升级已完成，但其间出现错误。"

#: ../DistUpgrade/DistUpgradeController.py:2013
msgid "System upgrade is complete."
msgstr "系统升级完成"

#: ../DistUpgrade/DistUpgradeController.py:2062
msgid "The partial upgrade was completed."
msgstr "部分升级完成。"

#: ../DistUpgrade/DistUpgradeController.py:2091
msgid "Lock screen disabled"
msgstr "锁屏已禁用"

#: ../DistUpgrade/DistUpgradeController.py:2092
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr "在重启之前您的锁屏功能已被禁用。"

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "无法找到发行注记"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "服务器可能已过载。 "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "无法下载发行说明"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherCore.py:75
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "请检查您的互联网连接。"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:70
#, fuzzy
msgid "Could not find the release announcement"
msgstr "无法找到发行注记"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:71
#, fuzzy
msgid "The server may be overloaded."
msgstr "服务器可能已过载。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:74
#, fuzzy
msgid "Could not download the release announcement"
msgstr "无法下载发行说明"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:80
#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "继续 [yN] "

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeFetcherCore.py:82
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "y"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:99
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "使用 '%(signature)s' 对 '%(file)s' 进行验证 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:125
#, python-format
msgid "extracting '%s'"
msgstr "正在提取 '%s'"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:146
#: ../DistUpgrade/DistUpgradeFetcherCore.py:147
msgid "Could not run the upgrade tool"
msgstr "不能运行升级工具"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:148
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"这很有可能是升级工具中的一个 bug。请使用命令“ubuntu-bug ubuntu-release-"
"upgrader-core”将其报为 bug。"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:225
msgid "Upgrade tool signature"
msgstr "升级工具签名"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:232
msgid "Upgrade tool"
msgstr "升级工具"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:266
msgid "Failed to fetch"
msgstr "下载失败"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:267
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "获取升级信息失败。可能网络有问题。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid "Authentication failed"
msgstr "认证失败"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:272
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "认证升级信息失败。可能是网络或服务器的问题。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
msgid "Failed to extract"
msgstr "提取失败"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:278
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "提取升级信息失败。可能是网络或服务器的问题。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:283
msgid "Verification failed"
msgstr "验证失败"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:284
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "验证升级程序失败。可能是网络或服务器的问题。 "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:298
#: ../DistUpgrade/DistUpgradeFetcherCore.py:304
msgid "Can not run the upgrade"
msgstr "不能运行升级"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:299
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"可能的原因是系统的 /tmp 目录无可执行权限。请以可执行权限重新挂载该目录，重新"
"升级。"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:305
#, python-format
msgid "The error message is '%s'."
msgstr "错误信息是“%s”。"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "升级"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "取消(&C)"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "发行注记"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "升级"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "正在下载附加软件包..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "文件 %s/%s 速度： %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "文件 %s/%s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "请将 '%s' 插入光驱 '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "改变介质"

#: ../DistUpgrade/DistUpgradeQuirks.py:230
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr "您的图形硬件可能无法在 Ubuntu 14.04 上完全支持。"

#: ../DistUpgrade/DistUpgradeQuirks.py:232
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""
"您的图形硬件不完全支持运行 'unity' 桌面环境。升级可能会产生一个非常缓慢的环"
"境。我们建议暂时保持 LTS 版本。要获取更多信息，请访问 https://wiki.ubuntu."
"com/X/Bugs/UpdateManagerWarningForUnity3D 您是否要继续升级？"

#: ../DistUpgrade/DistUpgradeQuirks.py:256
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr "Ubuntu 12.04 LTS 可能无法完全支持您的显卡。"

#: ../DistUpgrade/DistUpgradeQuirks.py:258
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"Ubuntu 12.04 LTS 对 Intel 图形硬件的支持有限制，升级后您可能会遭遇问题。更多"
"信息，请查看 https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx 。仍"
"然想要继续升级吗？"

#: ../DistUpgrade/DistUpgradeQuirks.py:278
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr "升级过程可能会降低桌面特效，游戏性能以及对显示要求高的程序。"

#: ../DistUpgrade/DistUpgradeQuirks.py:282
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"此计算机正在使用 AMD “fglrx”显卡驱动。该驱动在 Ubuntu 10.04 LTS 中没有与您的"
"硬件相适应的版本。\n"
"\n"
"您想继续吗？"

#: ../DistUpgrade/DistUpgradeQuirks.py:321
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:322
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:337
msgid "No ARMv6 CPU"
msgstr "没有 ARMv6 CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:338
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"您的系统使用的是比 ARMv6 架构旧的 ARM CPU。karmic 中所有软件包构建时进行的优"
"化都需要至少 ARMv6 的 CPU 架构。在这种硬件基础上无法将您的系统升级到一个新的 "
"Ubuntu 发行版。"

#: ../DistUpgrade/DistUpgradeQuirks.py:358
msgid ""
"Sorry, this storage driver is not supported in kernels for newer releases"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:360
#, python-format
msgid ""
"There will not be any further Ubuntu releases that provide kernel support "
"for the aufs storage driver.\n"
"\n"
"Please ensure that none of your containers are using the aufs storage "
"driver, remove the directory %s and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:380
msgid "No init available"
msgstr "无可用的 init"

#: ../DistUpgrade/DistUpgradeQuirks.py:381
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"您的系统似乎运行在一个没有 init daemon 的虚拟化环境中(如 Linux-VServer)，"
"Ubuntu 10.04 LTS 无法在这种环境中运行，您需要先更新您的虚拟机配置。\n"
"\n"
"您确定要继续吗？"

#: ../DistUpgrade/DistUpgradeQuirks.py:422
msgid "The Robot Operating System (ROS) is installed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:423
msgid ""
"It appears that ROS is currently installed. Each ROS release is very strict "
"about the versions of Ubuntu it supports, and Ubuntu upgrades can fail if "
"that guidance isn't followed. Before continuing, please either uninstall "
"ROS, or ensure the ROS release you have installed supports the version of "
"Ubuntu to which you're upgrading.\n"
"\n"
"For ROS 1 releases, refer to REP 3:\n"
"https://www.ros.org/reps/rep-0003.html\n"
"\n"
"For ROS 2 releases, refer to REP 2000:\n"
"https://www.ros.org/reps/rep-2000.html\n"
"\n"
"Are you sure you want to continue?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:554
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "与 Snap Store 的连接失败"

#: ../DistUpgrade/DistUpgradeQuirks.py:555
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:570
msgid "Connection to Snap Store failed"
msgstr "与 Snap Store 的连接失败"

#: ../DistUpgrade/DistUpgradeQuirks.py:571
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"您的系统无法连接到 Snap Store。为了获得最佳的升级体验，请确保您的系统可以连"
"接 api.snapcraft.io。\n"
"你还想继续升级吗？"

#: ../DistUpgrade/DistUpgradeQuirks.py:580
msgid "Outdated snapd package"
msgstr "过时的 snapd 软件包"

#: ../DistUpgrade/DistUpgradeQuirks.py:581
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:604
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:635
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:641
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:645
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:648
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:677
msgid "PAE not enabled"
msgstr "未启用 PAE 功能"

#: ../DistUpgrade/DistUpgradeQuirks.py:678
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"您的系统使用的 CPU 没有启用 PAE (物理地址扩展)功能。Ubuntu 至多支持非 PAE 功"
"能到 12.04 版。如果想升级到更高版本的 Ubuntu，您必须启用 PAE 功能(在可能的前"
"提下)。请常见：http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeQuirks.py:1050
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:62
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"使用前端。当前可用的有： \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:65
msgid "*DEPRECATED* this option will be ignored"
msgstr "*已废弃* 这个选项将被忽略"

#: ../DistUpgrade/DistUpgradeMain.py:68
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "仅执行部分升级(不重写 sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:71
msgid "Disable GNU screen support"
msgstr "关闭对 GNU screen 的支持"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Set datadir"
msgstr "设置数据目录"

#: ../DistUpgrade/DistUpgradeMain.py:76
msgid "Upgrade to the development release"
msgstr "升级到开发版"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "下载完成"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "正在下载文件 %li/%li 速度 %s/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "大约还要 %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "下载第 %li 个文件(共 %li 个文件)"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "正在应用更改"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "依赖关系问题 - 保持未配置状态"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "无法安装“%s”"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr "升级将继续进行，但“%s”可能没有工作。请考虑提交关于它的错误报告。"

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"替换自定义配置文件\n"
"“%s”吗？"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr "如果选择替换为新版本的配置文件，您将会失去所有已做的修改。"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "找不到 diff 命令"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "出现致命错误"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"请报告这个错误（如果还没有的话）并在报告中包括文件 /var/log/dist-upgrade/"
"main.log 和 /var/log/dist-upgrade/apt.log 。升级已取消。\n"
"您的原始 sources.list 已保存在 /etc/apt/sources.list.distUpgrade 。"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Crtl+C 被按下"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr "这将取消本次操作且可能使系统处于被破坏的状态。您确定要这样做？"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "为防止数据丢失，请关闭所有打开的应用程序和文档。"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "不再被 Canonical 支持 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>降级 (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "卸载 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "不再需要 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "安装 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "升级 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "显示差别 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< 隐藏差别"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "错误"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "关闭(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "显示终端 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< 隐藏终端"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "信息"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "开始升级"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "细节"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "不再支持 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "移除 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "移除(被自动安装的) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "安装 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "升级 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "需要重启"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>重新启动系统以完成升级</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "现在重启(_R)"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>取消正在运行的升级？</big></b>\n"
"\n"
"如果取消升级系统可能不稳定，强烈建议您继续升级。"

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "取消升级？"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li 天"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li 小时"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li 分钟"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li 秒"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"使用 1Mbit 的 DSL 连接下载需要大约 %s 秒的时间， 使用 56K 的调制解调器连接大"
"约需要 %s 秒时间。"

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "根据您的连接速度，这次下载将要用大约 %s 的时间 "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "正在准备升级"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "获得新的软件源"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "获取新的软件包"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "正在安装升级"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "正在清理"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d 个已安装的软件包不再被 Canonical 支持。您仍然可以获得社区支持。"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "将删除 %d 个软件包。"

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "将安装 %d 个新的软件包。"

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "将升级 %d 个软件包。"

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"您共需下载 %s。 "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr "安装升级可能会耗费几小时的时间。一旦下载完毕就不能取消该进程。"

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"升级文件的获取和安装可能会耗费几小时的时间。一旦下载完毕就不能取消该进程。"

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "包的删除可能会耗费几小时的时间。 "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "此计算机中的软件是最新软件。"

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "你的系统没有可用升级。升级被取消。"

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "需要重启"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "升级已经完成并需要重启。您要现在重启么？"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "进度：[%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"请报告这个错误并在报告中包括文件 /var/log/dist-upgrade/main.log 和 /var/log/"
"dist-upgrade/apt.log 。升级已取消。\n"
"您的原始 sources.list 已保存在 /etc/apt/sources.list.distUpgrade 。"

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "中止"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "降级：\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "按 [ENTER] 键以继续"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "详细信息[d]"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "不再支持：%s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "移除：%s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "安装：%s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "升级：%s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "继续 [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"为了完成升级，必须重新启动。\n"
"如果你选择“是”，系统将会重新启动。"

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "正在下载文件 %(current)li/%(total)li 速度 %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "正在下载文件 %(current)li/%(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "显示单个文件进度"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "取消升级(_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "恢复升级(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>取消正在运行的升级？</big></b>\n"
"如果您取消升级，系统可能不稳定。强烈建议您继续升级。"

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "开始升级(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "替换(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "文件之间的差别"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "报告 Bug(_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "继续(_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>开始升级？</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>重启系统以完成升级</big></b>\n"
"\n"
"请在继续前保存你的工作。"

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "发行版升级"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr "<b><big>正将Ubuntu升级至%s版</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "设定新的软件源"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "系统正在重新启动"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "终端"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "升级(_U)"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>有新版本的 Ubuntu 可用。您想升级吗？</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "不升级"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "稍后询问"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "是的，现在就升级"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "您已经拒绝升级到新版本的 Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr "您可以稍后升级，打开软件更新器并点击“升级”即可。"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "执行版本升级"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "升级 Ubuntu 版本前，您需要授权该操作。"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "执行部分升级"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "进行部分升级前，您需要授权该操作。"

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "显示版本并退出"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "包含数据文件的文件夹"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "运行指定的前端"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "运行部分升级"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "正在下载发布升级工具"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr "如果使用最新的支持版本，则升级到开发版本"

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:188
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "尝试通过 $distro-proposed 更新到最新版本。"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"以特定模式升级。\n"
"目前支持：用“桌面”为桌面系统，“服务器”为服务器系统升级。"

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr "仅在有新的发行版发布时检查，并通过退出码(exit code)报告结果"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr "--devel-release 与 --proposed 选项"

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr "不相容，请只用其中的一个选项。"

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr "正在检查新版 Ubuntu"

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "不再提供您的 Ubuntu 版本的支持。"

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"要获得关于升级的信息，请访问：\n"
"%(url)s\n"

#: ../do-release-upgrade:155
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:156
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:158
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:159
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:160
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:162
msgid "Upgrades to the development release are only "
msgstr "只能从受支持的最新版本 "

#: ../do-release-upgrade:163
msgid "available from the latest supported release."
msgstr "在最新的支持版本上可用。"

#: ../do-release-upgrade:165
msgid "No new release found."
msgstr "未找到新版本。"

#: ../do-release-upgrade:170
msgid "Release upgrade not possible right now"
msgstr "版本升级目前不可用"

#: ../do-release-upgrade:171
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr "发行版目前无法更新，请再试一次。服务器报告：'%s'"

#: ../do-release-upgrade:177
#, c-format
msgid "New release '%s' available."
msgstr "有新版本“%s”可供使用"

#: ../do-release-upgrade:178
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "运行“do-release-upgrade”来升级到新版本。"

#: ../do-release-upgrade:195
msgid "Please install all available updates for your release before upgrading."
msgstr "请在升级前安装您的发行版所有可用更新。"

#: ../do-release-upgrade:211
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:88
msgid "Software Updater"
msgstr "软件更新器"

#: ../check_new_release_gtk.py:120
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Ubuntu %(version)s 升级可用"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:151
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "您已经拒绝升级到 Ubuntu %s"

#: ../check_new_release_gtk.py:184
msgid "Check if upgrading to the latest devel release is possible"
msgstr "验证是否能够升级到最新版本"

#: ../check_new_release_gtk.py:193
msgid "Add debug output"
msgstr "添加调试输出"

#, fuzzy
#~ msgid "An upgrade is not possible at this time"
#~ msgstr "版本升级目前不可用"

#~ msgid "Could not calculate sources.list entry"
#~ msgstr "无法计算 sources.list 条目"

#~ msgid ""
#~ "Unable to locate any package files, perhaps this is not a Ubuntu Disc or "
#~ "the wrong architecture?"
#~ msgstr "无法定位任何软件包文件，也许这张不是 Ubuntu 光盘，或者其架构错误？"

#~ msgid "Failed to add the CD"
#~ msgstr "添加 CD 失败"

#~ msgid ""
#~ "There was a error adding the CD, the upgrade will abort. Please report "
#~ "this as a bug if this is a valid Ubuntu CD.\n"
#~ "\n"
#~ "The error message was:\n"
#~ "'%s'"
#~ msgstr ""
#~ "添加 CD 时出错，升级中止。如果这是一张有效的 Ubuntu CD，请您报告这个错"
#~ "误。\n"
#~ "\n"
#~ "错误信息是：\n"
#~ "'%s'"

#~ msgid "Include latest updates from the Internet?"
#~ msgstr "包括网络上的最新更新？"

#~ msgid ""
#~ "The upgrade system can use the internet to automatically download the "
#~ "latest updates and install them during the upgrade.  If you have a "
#~ "network connection this is highly recommended.\n"
#~ "\n"
#~ "The upgrade will take longer, but when it is complete, your system will "
#~ "be fully up to date.  You can choose not to do this, but you should "
#~ "install the latest updates soon after upgrading.\n"
#~ "If you answer 'no' here, the network is not used at all."
#~ msgstr ""
#~ "升级系统能够从网络上自动下载最新的更新文件并自动安装。如果网络连接可用，强"
#~ "烈建议启用此选项。\n"
#~ "\n"
#~ "升级时间可能加长，但升级结束后，您的系统将是最新的。您也可以选择在升级结束"
#~ "后再进行系统更新。\n"
#~ "如果选择\"否\"，就不会使用网络。"

#~ msgid "Failed to add the cdrom"
#~ msgstr "添加光驱失败"

#~ msgid "Sorry, adding the cdrom was not successful."
#~ msgstr "对不起，没有成功添加光驱。"

#~ msgid "Use the given path to search for a cdrom with upgradable packages"
#~ msgstr "使用所给的路径查找带升级包的 CD-ROM"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "在准备升级时发生了一个无法解决的问题：\n"
#~ "%s\n"
#~ "\n"
#~ " 这可能是由以下原因引起的：\n"
#~ " * 升级到了预发行 Ubuntu 版本\n"
#~ " * 正在运行当前的预发行 Ubuntu 版本\n"
#~ " * 非 Ubuntu 提供的非官方软件包\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "您的系统没有安装 ubuntu-desktop，kubuntu-desktop 或 eubuntu-desktop 软件包"
#~ "所以无法确定运行的 ubuntu 的版本。\n"
#~ " 请先用新立得或 APT 安装以上所举软件包中的一个。"

#~ msgid "Sandbox setup failed"
#~ msgstr "安装沙盒失败"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "不能创建沙盒环境"

#~ msgid "Sandbox mode"
#~ msgstr "沙盒模式"

#~ msgid ""
#~ "This upgrade is running in sandbox (test) mode. All changes are written "
#~ "to '%s' and will be lost on the next reboot.\n"
#~ "\n"
#~ "*No* changes written to a system directory from now until the next reboot "
#~ "are permanent."
#~ msgstr ""
#~ "本次更新运行在沙盘(测试)模式，所有变更都将写入 '%s' 并会在重启后丢失。\n"
#~ "\n"
#~ "从现在起对系统目录的变更重启后都将 *不复存在* 。"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr "您的 python 安装错误，请修复“/usr/bin/python”符号链接。"

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "已安装软件包“debsig-verify”"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "由于安装了上述软件包而无法继续升级。\n"
#~ "请使用新立得软件包管理器来移除它，或者先使用 “sudo apt-get remove debsig-"
#~ "verify”卸载后再重新尝试升级。"

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "扫描仓库时未发现升级信息。可能您使用的是内部镜像或者镜像信息已经过期。\n"
#~ "\n"
#~ "是否依然重写 \"sources.list\" 文件？如果选择“是”，将会把所有 “%s” 升级到 "
#~ "“%s”。"

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "升级已被中断。此次升级需要有 %s 的可用空间在磁盘 %s 上。请释放至少 %s 的空"
#~ "间在磁盘 %s 上。您可以清空回收站并使用“sudo apt-get clean”命令以清除之前安"
#~ "装操作留下的临时文件。"

#~ msgid "evms in use"
#~ msgstr "evms 使用中"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "您的系统在 /proc/mounts 中使用“evms”卷管理器。“evms”软件不再被支持，请关闭"
#~ "它并重新运行升级。"

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "此计算机正在使用 NVIDIA “nvidia”显卡驱动。该驱动在 Ubuntu 10.04 LTS 中没有"
#~ "与您的硬件相适应的版本。\n"
#~ "\n"
#~ "您想继续吗？"

#~ msgid "No i686 CPU"
#~ msgstr "非 i686 CPU"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "您的系统使用的是 i586 或是没有“cmov”扩展的 CPU。所有优化生成的软件包都需要"
#~ "最低 i686 的架构。这样的硬件无法升级到新的 Ubuntu 发行版。"

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "使用 aufs 进行沙盒升级"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "进行带沙盒 aufs 隔离层的测试升级"

#~ msgid "No new release found"
#~ msgstr "未发现新版本"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "执行版本升级需要授权"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "执行部分升级需要授权"

#~ msgid ""
#~ "If none of this applies, then please report this bug using the command "
#~ "'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
#~ msgstr ""
#~ "如果这些都不生效，请在终端中使用命令 'ubuntu-bug ubuntu-release-upgrader-"
#~ "core' 来报告 bug。"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "在筹划升级时，一个不可解决问题发生了：\n"
#~ "\n"
#~ " 这可能是由以下原因造成的：\n"
#~ " *升级为 Ubuntu 的预发行版本\n"
#~ " *运行目前的 Ubuntu 预发布版本\n"
#~ " *不是由 Ubuntu 提供的非官方软件包\n"
#~ "\n"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
#~ msgstr "<b><big>正将Ubuntu升级至15.10版</big></b>"

#~ msgid "apt architecture is i386"
#~ msgstr "apt 架构为 i386"

#~ msgid ""
#~ "Upgrades from 18.04 on the i386 architecture are not supported at this "
#~ "time."
#~ msgstr "目前还不支持在 i386 架构上从 18.04 升级系统。"
